assets/js/util.js

```diff
@@ -119,6 +122,28 @@ $(document).ready(function () {
     $('.modal').on('hide.bs.modal', function () {
         $('a[data-target=#' + $(this).attr('id') + ']').parents('li').removeClass('active');
     });
+
+    /*  Following is the function that provides functionality of back-to-top button */
+
+    $(window).scroll(function () {
+        if($(window).scrollTop() > amountScrolled) {
+            $('#backToTop').removeClass('hidden-xs hidden-sm hidden-md hidden-lg');
+            $('#backToTop').addClass('hidden-xs hidden-sm visible-md visible-lg');
+        }
+        else {
+            $('#backToTop').removeClass('hidden-xs hidden-sm visible-md visible-lg');
+            $('#backToTop').addClass('hidden-xs hidden-sm hidden-md hidden-lg');
+        }
+    });
+
+    $('#backToTop').click(function () {
+        $('html, body').animate({
+            scrollTop: 0
+        }, transitionTime);
```

Comment:
@sushain97  I will remove the var `transitionTime` from the code and just put a `fast` there!